api/php.ini

```diff
@@ -144,7 +144,7 @@ session.referer_check =
 session.cache_limiter = nocache
 session.cache_expire = 180
 session.use_trans_sid = 0
-session.sid_length = 48
+session.sid_length = 64
```

Comment:
Why not max it out to the maximum possible value (256) to increase the security level?
"[Longer session ID is harder to guess](https://www.php.net/manual/en/session.configuration.php#ini.session.sid-length)".  🤗